src/metabase/embed/settings.clj

```diff
@@ -44,8 +44,29 @@
                                           :number-embedded-questions  (t2/count :model/Card :enable_embedding true)
                                           :number-embedded-dashboards (t2/count :model/Dashboard :enable_embedding true)}))))
 
+(defsetting enable-embedding-static
+  (deferred-tru "Allow admins to embed Metabase via static embedding?")
+  :type       :boolean
+  :default    false
+  :visibility :authenticated
+  :export?    false
+  :audit      :getter
+  :setter     (fn [new-value]
+                (when (not= new-value (setting/get-value-of-type :boolean :enable-embedding-static))
+                  (setting/set-value-of-type! :boolean :enable-embedding-static new-value)
+                  (when (and new-value (str/blank? (embed/embedding-secret-key)))
+                    (embed/embedding-secret-key! (crypto-random/hex 32))))))
```

Comment:
This copies the value from the old `enable-embedding` setting minus the snowplow part.